modules/primer-utilities/docs/typography.md

```diff
@@ -93,6 +93,7 @@ Change the font weight, styles, and alignment with these utilities.
 <p class="text-emphasized">Emphasized</p>
 <p class="text-small">Small</p>
 <p class="lead">Bacon ipsum dolor amet tri-tip chicken kielbasa, cow swine beef corned beef ground round prosciutto hamburger porchetta sausage alcatra tail.</p>
+<p class='text-mono'>Monospace</p>
```

Comment:
You'll need to use double quotes here sorry, it's not jsx 😆

Reply:
😂 good catch! thanks :)